security/policies-and-procedures.md

```diff
@@ -0,0 +1,24 @@
+# Policies and Procedures
+
+Currently the team is working to finalize this work and have it completed for the 1.8 Release and for the transition to the CNCF.
+These policies and procedures will be based of the [Kubernetes Policy](https://kubernetes.io/docs/reference/issues-security/security/)
+We are working on updating and editing these policies and procedures for the Kubeflow Community.  Our very **rough DRAFT and working docs** can be found [here](https://docs.google.com/document/d/1sjWa0G2UqdsP1QROYVEl9iQR6v1x8HD7-F4TQo7lV6M/edit?usp=sharing)
+
+## The first iteration of the Kubeflow Security Policies and Procedures are as follows*:
+
+*Please note we will continue to iterate on this with each release and as we get commitments from our distributions and volunteers from each of the Kubeflow components and working groups (WG). As the team grows we will work to set up roles and responsibilities to bring us closer in line with the Kubernetes policies and procedures.* 
+
+The Kubeflow Security team (listed below as “we”) will take a multi-level, transparent approach to the security of Kubeflow (product/code). At this time we will base this effort to align with each release.  As noted above as we get more volunteers to help with these efforts we will continue to iterate on the cadence.  
+
+1. We will encourage Kubeflow users to open issues with the respective WG that the vulnerability was found and tag it with the “security” tag. If they don’t know how to do that or don’t feel comfortable doing so, they can also contact us via our #security slack channel.  And if a private mailing list/group is needed we will set that up. 
```

Comment:
It is strongly recommended that we don't add a security tag to an open issue. 

You can find a security template put up by CNCF in https://github.com/cncf/tag-security/tree/main/project-resources.

Reference: https://github.com/cncf/tag-security/blob/main/project-resources/templates/SECURITY.md#reporting-a-vulnerability

Most CNCF projects right now use email to notify security issue. GitHub just released their private vuln reporting feature (https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing/privately-reporting-a-security-vulnerability). Either email shared among the people who will be responsible for vuln response or this new GitHub reporting channel are fine.

Reply:
@zijianjoy, I agree, but when I asked about getting an email created for the security team there was some push back. Since Google has to create the email address can you all create a security@kubeflow.org email for the team. The Working Groups asked that security issues related respective working groups be filed their, so (at the time) without being able to create an official email for the group and the resources we had available we came up with the above process. I'll take a look at the resources you have listed and rework the process and bring it up to the group on the 24th at our next meeting. 

In order to get this PR approved if I change this file to say we are currently working on the process and a link to our meeting notes would this work for sake of approving this?

Reply:
We are in discussion about creating such email address. Make sense to track this effort in another PR. At the meantime, James Wu has LGTMed your PR here.